src/client.rs

```diff
@@ -143,7 +143,7 @@ impl Client {
     pub async fn execute<'a, 'b>(
         &'a mut self,
         query: impl Into<Cow<'b, str>>,
-        params: &'b [&'b dyn ToSql],
+        params: &'b [&'b (dyn ToSql + Send + Sync)],
```

Comment:
Wouldn't defining `ToSql` as `ToSql: Send + Sync` instead be easier?

Reply:
Yeah, I guess it would be.